main.py

```diff
@@ -70,8 +76,10 @@ def create_pr(self, message, branch_name, labels):
 @click.command()
 @click.option('-h', '--ghe-hostname', type=str, default="", help='Github Enterprise Hostname, example: \'github.company.com\'')
 @click.option('-o', '--organization', type=str, required=True)
-@click.option('-f', '--find', type=str, required=True)
-@click.option('-r', '--replace', type=str, required=True)
+@click.option('-f', '--find', type=str, required=True, help='The string that is going to be searched in GH.')
+@click.option('-g', '--find_file', type=str, required=True, help='The string or regex pattern that is going to be replaced inside the matching files.')
```

Comment:
```suggestion
@click.option('-s', '--match-string', type=str, required=False, help='Optional string or regex pattern that is going to be replaced inside the matching files instead of --find')
```